src/containers/FactoryLoader.tsx

```diff
@@ -184,12 +197,13 @@ export class FactoryLoaderContainer extends React.PureComponent<Props, State> {
       this.updateOverrideParams(key, val);
       factoryUrl.searchParams.append(key, val);
     });
-
     attrs.stackName = factoryUrl.toString();
+    const createPolicy = attrs['policies.create'] as CreatePolicy || 'perclick';
```

Comment:
These attributes are inputs we should validate. I guess you can create and use a type guard function for this purpose.

Reply:
Ok. I have fixed it.